autoblocks/_impl/prompts/manager.py

```diff
@@ -32,6 +35,32 @@
 MinorVersionEnumType = TypeVar("MinorVersionEnumType", bound=Enum)
 
 
+def is_testing_context() -> bool:
+    """
+    Note that we check for the presence of the CLI environment
+    variable and not the test case contextvars because the
+    contextvars aren't set until run_test_suite is called,
+    whereas a prompt manager might have already been imported
+    and initialized by the time run_test_suite is called.
+    """
+    return bool(AutoblocksEnvVar.CLI_SERVER_ADDRESS.get())
```

Comment:
Should we just check for the prompt snapshots env variable?

Reply:
i'm using this to know if we're in a testing context at all (even if prompt snapshots are not set) to disable the latest refresh, since i don't think it makes sense in a testing context since you probably want the prompt to be stable during a test run

Reply:
https://github.com/autoblocksai/python-sdk/pull/149/files#diff-4e124f39d32018a40189c28c9452a15f042fd1c6eb54d58eeb286ed42c8b6381R122

Reply:
ahh i see